test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -3009,6 +3009,197 @@ public async Task TestPackageManager_PreviewProjectsUninstallPackageAsync_Uninst
             }
         }
 
+        [Fact]
+        public async Task GetTransitivePackagesAsync_WithTransitivePackageReferences_ReturnsPackageIdentities()
+        {
+            using (var testDirectory = TestDirectory.Create())
```

Comment:
Some of the things below, particularly creating the global packages folder and a package source folder, gets done automatically in https://github.com/NuGet/NuGet.Client/blob/dev/test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestPathContext.cs, plus it creates a nuget.config with it all configured.

Not worth changing now, but FYI.

Reply:
I wasn't aware of that. I'll keep it in mind for next time.